pgmpy/factors/CPD.py

```diff
@@ -116,15 +119,33 @@ def __init__(self, variable, variable_card, values,
 
         super(TabularCPD, self).__init__(variables, cardinality, values.flatten('C'))
 
-    def get_cpd(self):
+    def get_cpd(self, pretty=True):
         """
         Returns the cpd
         """
         if self.variable in self.variables:
+            if pretty:
+                table_list = []
+                for i in range(len(self.cardinality))[1:]:
+                    var_rep = np.prod(self.evidence_card[i:])
+                    row = [self.scope()[-i]]
+                    row.extend(self.variables[row[0]] * var_rep)
+                    if i > 1:
+                        row[1:] = [x for item in row[1:] for x in
+                                   repeat(item, int(np.prod(self.evidence_card[0:i - 1])))]
+                    table_list.insert(0, row)
+
+                value_list = self.values.reshape(self.cardinality[0], np.prod(self.cardinality[1:])).tolist()
+                for item, grade in zip(value_list, self.variables[self.variable]):
+                    item.insert(0, grade)
```

Comment:
@kislayabhi: Check `numpy.column_stack`. You should be able to replace this loop with a single statement.